packages/go/openapi/src/schemas/model.saved-queries-permissions.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+allOf:
+  - $ref: './model.components.int64.id.yaml'
+  - $ref: './model.components.timestamps.yaml'
+  - type: object
+    properties:
+      id: 
+        type: integer
+        format: int64
```

Comment:
This property is actually covered by the model.components.int64.id part above. i suspect that this may have happened because the SavedQueryPermissions model has an ID field defined, though it would probably be better for that model to remove its ID field and replace the `Basic` embedded struct with a `BigSerial`, which has all of Basic + an int64 ID